terraform-common.mk

```diff
@@ -47,4 +47,4 @@ graph:
 	terraform graph -draw-cycles | dot -Tpng > graph.png
 
 $(ENV_NAME).tfvars:
-	$(TOP)/bin/generate-tfvars >$@
+	$(TOP)/bin/generate-tfvars $@
```

Comment:
Oh, good point, I missed that this was Make and not bash, :man_facepalming:. Not sure how Make handles `$@` and spaces and all, but since I don't think `$(ENV_NAME)` will have spaces in it(?) then I don't think it's really much of an issue.